src/sdl2/video.rs

```diff
@@ -1399,6 +1429,24 @@ impl Window {
         }
     }
 
+    #[doc(alias = "SDL_GetWindowICCProfile")]
+    pub fn icc_profile(&self) -> Result<Vec<u8>, String> {
+        unsafe {
+            let mut size: sys::size_t = 0;
+            let data = sys::SDL_GetWindowICCProfile(
+                self.context.raw,
+                &mut size as *const sys::size_t as *mut _,
```

Comment:
doesn't `&mut size as *mut _` work here? I'm not sure casting a *const to a *mut is defined behavior, even within unsafe rust.

Reply:
Fixed